pkg/option/config.go

```diff
@@ -994,6 +994,10 @@ const (
 	// HubbleMetrics specifies enabled metrics and their configuration options.
 	HubbleMetrics = "hubble-metrics"
 
+	// HubbleFlowlogsConfigFilePath specifies the filepath with configuration of hubble flowlogs.
+	// e.g. "/etc/cilium/flowlog.yaml"
+	HubbleFlowlogsConfigFilePath = "hubble-flowlogs-config"
```

Comment:
nit: Is there any naming convention here? It seems like some other items have `-path` suffix to indicate directory/file (as opposed to KV store/ConfigMap, etc.)